lib/packwerk/checker.rb

```diff
@@ -32,5 +53,23 @@ def invalid_reference?(reference); end
 
     sig { abstract.params(reference: Reference).returns(String) }
     def message(reference); end
+
+    sig { returns(TodoLocation) }
+    def todo_location
+      TodoLocation::PackageReferencingConstant
+    end
+
+    sig { params(reference: Reference).returns(Package) }
+    def todo_file_for(reference)
```

Comment:
```suggestion
    def package_for(reference)
```

Reply:
Incorporated this as `offending_package_for`